references/oidc-mock/test/integration/features/oidc-mock.certs.feature

```diff
@@ -0,0 +1,18 @@
+@CertsEndpoint
+Feature:
+  As a Client App 
+  I want to get JWKS keys 
+  So that I can validate JWT token (id_token)
+
+  Scenario: Client App Accesses JWKS Resource
+    When I GET /certs
+    Then response code should be 200
+    And response body path $.keys[0].alg should be (RS256)
```

Comment:
do you need the brackets `()` here? it is doing a regex match so shouldnt be necessary